src/raft_server.cxx

```diff
@@ -1047,6 +1047,7 @@ void raft_server::become_leader() {
             // reconnect_client(*pp);
 
             pp->set_next_log_idx(log_store_->next_slot());
+            pp->reset_stream();
```

Comment:
Shall we need to also disable the streaming when it become follower?

Reply:
Yes, makes sense.